HealthSDK/GiniHealthSDK/Documentation/source/Testing.md

```diff
@@ -89,12 +89,33 @@ The following is an example for the url `gini-pay://payment-requester`:
                      open url: URL,
                      options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
         if url.host == "payment-requester" {
+            processBankUrl(url: url)
             // hadle incoming url from the banking app
         }
         return true
     }
 ```
 
+Here you can optain the paymentRequestId and check the payment status:
```

Comment:
[request] Here should be `obtain` instead of `optain`. should we mark paymentRequestId inside of `paymentRequestId ` ?